### PR TITLE
Update name everywhere

### PR DESCRIPTION
**Pull request by marksiemers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,8 +17,8 @@ Add this to your application's `shard.yml`:
 
 ```yaml
 dependencies:
-  amber_spec:
-    github: amberframework/amber-spec
+  garnet-spec:
```
Comment (faustinoaq) [inline README.md:20]: We should use `garnet_spec` here, we already have `jasper_helper`, `granite_orm` and `quartz_mailer`

File: README.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ If you're running in a different OS such a Linux you can specify the chromedrive
 ```crystal
 # spec/spec_helper.cr
 require "amber"
-require "amber_spec"
+require "garnet-spec"
```
Comment (faustinoaq) [inline README.md:40]: :ditto: `garnet_spec`

File: shard.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-name: amber_spec
+name: garnet-spec
```
Comment (faustinoaq) [inline shard.yml:1]: :ditto: `garnet_spec`

File: spec/spec_helper.cr [hunk 4]
```diff
@@ -1,5 +1,5 @@
 require "spec"
-require "../src/amber_spec"
+require "../src/garnet-spec"
```
Comment (faustinoaq) [inline spec/spec_helper.cr:2]: :ditto: `garnet_spec`

File: src/amber_spec.cr [hunk 5]
```diff
@@ -1,7 +1,7 @@
 require "spec"
-require "./amber_spec/system/**"
-require "./amber_spec/controller/*"
-require "./amber_spec/selenium_server"
+require "./garnet-spec/system/**"
+require "./garnet-spec/controller/*"
+require "./garnet-spec/selenium_server"
```
Comment (faustinoaq) [inline src/amber_spec.cr:4]: :ditto: `garnet_spec`

**Review by faustinoaq — COMMENTED** [review]
Good renaming, just some comment about using `garnet_spec` instead of `garnet-spec`

We're already using underscore for all amber dependencies and dashes for repository url

**Comment by marksiemers** [comment]
#4

**Pull request closed without merging**